Quick-Start/Anatomy_of_extension.md

```diff
@@ -1,418 +1,418 @@
+1.  **Extension metadata**: The Control file contains metadata about the extension, such as its name, version, description, author, and license. This information helps users and administrators understand the purpose and characteristics of the extension. 
+ 
+2.  **Dependency management**: The Control file allows you to specify any dependencies that the extension has on other extensions or PostgreSQL versions. This ensures that the required dependencies are present before the extension can be installed or upgraded. 
+ 
+3.  **Installation instructions**: The Control file provides instructions on how to install the extension. It specifies the SQL script file(s) that need to be executed during installation, as well as any additional installation steps or considerations. 
+ 
+4.  **Upgrade instructions**: If the extension supports upgrading from a previous version, the Control file includes instructions on how to perform the upgrade process. This may involve executing SQL scripts to migrate data, modify existing objects, or handle any other necessary changes. 
+ 
+5.  **Uninstallation instructions**: The Control file defines the steps for uninstalling the extension. It specifies the SQL script file(s) or commands needed to remove the extension's objects and revert any changes made during installation. 
+ 
+6.  **Configuration options**: The Control file allows you to define custom configuration options for the extension. These options can be set by users or administrators to modify the behaviour or settings of the extension according to their needs. 
+ 
+7.  **Permissions and privileges**: The Control file can specify the required permissions and privileges that users must have to install, upgrade, or use the extension effectively. This helps ensure proper security and access control for the extension's functionality.  
+ 
+8.  **Documentation and reference**: The Control file often includes a section for additional documentation or references related to the extension. This can include links to external resources, tutorials, examples, or any other relevant information that helps users understand and utilize the extension effectively. 
+ 
+In summary, a Control file in PostgreSQL extensions serves as a crucial component that provides metadata, installation instructions, upgrade guidelines, configuration options, and other relevant information about the extension. It facilitates the installation, management, and usage of the extension in a PostgreSQL environment. 
+ 
+## Creating PostgreSQL extension: 
+When creating a PostgreSQL extension, writing a Control file is an essential step. The Control file provides crucial metadata and instructions for the installation, upgrade, and management of the extension. Here's a quick start guide on how to write a Control file for your PostgreSQL extension: 
+ 
+1.  **Choose a filename**: The Control file is typically named "control" or "your_extension_name.control". Make sure to use the ".control" file extension. 
```

Comment:
I believe it needs to be named `$extension_name.control`. Give [the docs](https://www.postgresql.org/docs/15/extend-extensions.html) a read for the precise requirements.